docs/config.md

```diff
@@ -90,16 +90,6 @@ version = 2
     # default_runtime_name is the default runtime name to use.
     default_runtime_name = "runc"
 
-    # 'plugins."io.containerd.grpc.v1.cri".containerd.default_runtime' is the runtime to use in containerd.
-    # DEPRECATED: use `default_runtime_name` and `plugins."io.containerd.grpc.v1.cri".runtimes` instead.
-    # Remove in containerd 1.4.
-    [plugins."io.containerd.grpc.v1.cri".containerd.default_runtime]
-
-    # 'plugins."io.containerd.grpc.v1.cri".containerd.untrusted_workload_runtime' is a runtime to run untrusted workloads on it.
-    # DEPRECATED: use `untrusted` runtime in `plugins."io.containerd.grpc.v1.cri".runtimes` instead.
-    # Remove in containerd 1.4.
-    [plugins."io.containerd.grpc.v1.cri".containerd.untrusted_workload_runtime]
-
```

Comment:
removing from here is fine... but the significant number of deletions do bring up a thought that maybe we need a doc or appendix for admins to help them transition from the deleted config items.